src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/Internal/HostTests.cs

```diff
@@ -1238,8 +1238,8 @@ public void BackgroundServiceAsyncExceptionGetsLogged()
                 })
                 .Start();
 
-            // give the background service 5 seconds to log the failure
-            Task timeout = Task.Delay(new TimeSpan(0, 0, 5));
+            // give the background service 1 minute to log the failure
+            Task timeout = Task.Delay(TimeSpan.FromMinutes(1));
```

Comment:
This is test code so we're more lenient, but this isn't a great use of Task.Delay, especially since this timer will likely end up running long after the test has completed.  It'd be better if this were just a simple Stopwatch, e.g.
```C#
Stopwatch sw = Stopwatch.StartNew();
while (true)
{
    ...
    Assert.InRange(sw.Elapsed, TimeSpan.Zero, TimeSpan.FromMinutes(1));
    await Task.Delay(TimeSpan.FromMilliseconds(30));
}
```

Reply:
Fixed. Thanks for the feedback.